src/Sulu/Bundle/ResourceBundle/Entity/FilterRepository.php

```diff
@@ -47,12 +47,13 @@ public function findByUserAndContextAndLocale($locale, $context, $userId)
             $qb = $this->getFilterQuery($locale);
             $qb->leftJoin('filter.user', 'user')
             ->andWhere('filter.context = :context')
-            ->orWhere('(user.id = :userId')
-            ->orWhere('filter.private = false)')
+            ->andWhere('(user.id = :userId OR filter.private = false)')
```

Comment:
Maybe we should have deleted that bundle before you struggled with it because of the tests :sweat_smile: